src/Uno.UI/UI/Xaml/UIElement.Pointers.cs

```diff
@@ -792,13 +792,18 @@ private protected virtual void PrepareShare(DataPackage data)
 
 		internal void RaiseDragEnterOrOver(global::Microsoft.UI.Xaml.DragEventArgs args)
 		{
-			var evt = IsDragOver(args.SourceId)
-				? DragOverEvent
-				: DragEnterEvent;
+			var wasDragOver = IsDragOver(args.SourceId);
 
 			(_draggingOver ??= new HashSet<long>()).Add(args.SourceId);
 
-			SafeRaiseEvent(evt, args);
+			if (!wasDragOver)
+			{
+				var args2 = new DragEventArgs(args);
+				SafeRaiseEvent(DragEnterEvent, args);
+				args.AcceptedOperation = args2.AcceptedOperation;
```

Comment:
Why creating a new `DragEventArgs` just to "save" the `AcceptedOperation` (you are not using it in the `SafeRaiseEvent` ?

I think this is enough:
```suggestion
				SafeRaiseEvent(DragEnterEvent, args);
				args.Handled = false;
```

Reply:
That's how WinUI does it. The 2 args are not the same. I do think it's unnecessary too (your suggestion was the initial implementation), so I'm on board if it's okay to not match WinUI in this instance.